android/gradle.properties

```diff
@@ -1,41 +1,7 @@
-# Project-wide Gradle settings.
 
-# IDE (e.g. Android Studio) users:
-# Gradle settings configured through the IDE *will override*
-# any settings specified in this file.
+TeadsSdkModule_kotlinVersion=1.7.0
```

Comment:
I see that these versions are different from the android sdk module:
https://github.com/ebuzzing/service-inapp-sdk-android/blob/f8c509fd09a063dc1665f6c7646379bf98c069f1/buildSrc/src/main/java/tv/teads/AndroidLibConfig.kt

```
    const val minSdk = 19
    const val targetSdk = 28
    [...]
```

I am wondering if it could impact in something 🤔